assets/sass/protocol/includes/_lib.scss

```diff
@@ -44,5 +44,5 @@ $type-scale: 'standard' !default;
 // Import all the things!
 @import './node_modules/@mozilla-protocol/tokens/dist/index';
 @import 'functions';
```

Comment:
**curious**: are we keeping the `get-theme` and `type-scale` functions in code for a while? I don't see them deleted (and I guess if they are not deleted, we don't have to make this a breaking change?)

Reply:
I didn't want to delete them, since it would introduce a breaking change - but if we should go all out and remove them here I'd be happy to do so. @craigcook

Reply:
Happy to leave as-is in this PR.

Reply:
It might be worth keeping them for a while and adding a deprecation notice like we've done with some components, to give some transition time for anyone using those functions. But I'm not sure how widely they're used outside of Protocol itself... we only use `get-theme` in a few places in bedrock.

Reply:
wonder how difficult it would be to write a migration script to do a search and replace (potentially with a flag setting for optional support query wrapping):

```scss
color: get-theme('title-text-color');
/* replace only */
color: var(--title-text-color);

@include bidi(((padding-left, get-theme('h-grid-sm'), padding-right, get-theme('h-grid-sm')),))
/* replace & use support query */
@support (--css: variables) {
    @include bidi(((padding-left, var(--h-grid-sm), padding-right, var(--h-grid-sm)),))
}
```

I'm on the fence about including the support queries in end user code. I think it's important for clarity in Protocol, but since we've set all the IE defaults already, it should be safe for users to directly set custom property values on their style rules. Browser parsers that don't support custom properties will ignore those rules and use the Protocol defaults.